09-schema/00-overview.md

```diff
@@ -85,7 +85,7 @@ In the sections that follow, by looking at various real-world examples, we learn
 In the following sections, we learn how to define a schema using Graql code in a `schema.gql` file. However, defining a schema can also be done programmatically (at runtime) using one of the Grakn Clients - [Java](../03-client-api/01-java.md#graql), [Python](../03-client-api/02-python.md#lazily-execute-a-graql-query) and [Node.js](../03-client-api/03-nodejs.md#lazily-execute-a-graql-query).
 
 ## Load the schema
-Once we have defined the schema, the next immediate step is to load it into Grakn. Learn how to [load the schema via the Grakn Console](../02-running-grakn/02-console.md#console-options).
+Once we have defined the schema, the next immediate step is to load it into Grakn. We can do this using the Grakn Console that comes in the Grakn distribution. In general, the command will look something like `grakn console -f <path to schema file`. The  [Grakn Console](../02-running-grakn/02-console.md#console-options) documentation provides further options.
```

Comment:
@flyingsilverfin  the command is missing the closing `>`.

also, the guidelines suggest that with rare exceptions we should always use the present tense. (`the command looks something like` over `the command will something like`). ignore this comment, if you believe this is one of those rare exceptions.

Reply:
ta